layers/meta-opentrons/recipes-bsp/u-boot/u-boot-distro-boot/boot.cmd.in

```diff
@@ -116,7 +128,7 @@ then
 else
     env set fdt_resize 'fdt addr ${fdt_addr_r} && fdt resize 0x20000'
     env set set_bootcmd_dtb 'env set bootcmd_dtb "echo Loading DeviceTree: \\${rootfs_boot_dir}/\\${fdtfile}; ${load_cmd} \\${fdt_addr_r} \\${rootfs_boot_dir}/\\${fdtfile}"'
-    env set set_apply_overlays 'env set apply_overlays "for overlay_file in \\${fdt_overlays}; do echo Applying Overlay: \\${rootfs_boot_dir}/\\${overlay_file} && ${load_cmd} \\${loadaddr} \\${overlays_prefix}\\${overlay_file} && fdt apply \\${loadaddr}; env set overlay_file; done; true"'
+    env set set_apply_overlays 'env set apply_overlays "for overlay_file in \\${fdt_overlays}; do env set skip 0; if test \${som_can_20hz} = 0 && test \\${overlay_file} = \${som_can_20hz_overlay}; then env set skip 1; fi; test \\${skip} = 0 && echo Applying Overlay: \\${rootfs_boot_dir}/\\${overlay_file} && ${load_cmd} \\${loadaddr} \\${overlays_prefix}\\${overlay_file} && fdt apply \\${loadaddr}; env set overlay_file; done; true"'
```

Comment:
No, because we always set it to `env set 0` when we are iterating over the overlay files.
That being said, your suggestion is much cleaner and simplifies the logic so I've made the change.